### PR TITLE
fixed walkToPoint

### LINKED ISSUE #33: Poke.player.walkToPoint not working
needs a fix

### PR DESCRIPTION
**Pull request by rohanp**
Fixes #33 

test
```
await Poke.player.walkToPoint(38.9176263,-77.4105144)
```
output
```
[i] Walking closer to [38.9176263,-77.4105144] - distance is: 177 meters
[i] Walking closer to [38.9176263,-77.4105144] - distance is: 165 meters
[i] Walking closer to [38.9176263,-77.4105144] - distance is: 153 meters
[i] Walking closer to [38.9176263,-77.4105144] - distance is: 141 meters
...
```

### CODE REVIEW

File: src/Player.js [hunk 1]
```diff
@@ -108,27 +108,39 @@ class Player {
...
+                }
+
     //distance less than 10 meters?
     if (distance <= 10){
       return true
     } else {
+
+      console.log(`[i] Walking closer to [`+lat+`,`+long+`] - distance is: ${distance} meters`)
+      await new Promise(resolve => setTimeout(resolve, 2000))
+      this.location  = newLocation
```
Comment (stoffern) [inline src/Player.js]: Set location before waiting..

**Comment by gallexme** [comment]
how fast is it walking then?

**Comment by rohanp** [comment]
6 m/s or ~15 mi/hr

**Comment by gallexme** [comment]
i know no human who can walk that fast

File: src/Player.js (hunk 1 above)
Comment (rohanp) [inline src/Player.js]: like this?

```
      this.location  = newLocation
      console.log(`[i] Walking closer to [`+lat+`,`+long+`] - distance is: ${distance} meters`)
      await new Promise(resolve => setTimeout(resolve, 2000))
      this.walkToPoint(lat, long)
```

**Comment by jimmywarting** [comment]
> i know no human who can walk that fast

Maybe he is riding a bicycle?

**Comment by stoffern** [comment]
@rohanp thanks for the PR, but we would need a random human walk..
As far as i can see you are splitting up the diff in 12 here
https://github.com/stoffern/pokemongo-api/pull/42/commits/cea248561dbece4b293e793d03d6270887d40c61?diff=unified#diff-57dfd10e73b05f2ac7dda337ab53d555R125


if it is 50 miles to target then you would move quite fast! :smile:

**Comment by rohanp** [comment]
ah turns out max speed to count towards egg hatching is 10 mi/hr. I'll lower this to 4 m/s. 

I can also add some stochasticity into it

**Comment by stoffern** [comment]
Problem fixed here https://github.com/stoffern/pokemongo-api/commit/8c111d2c6011db8b2df9757e72ca4fd8a15e4e86

walking random lat, long towards target (min 1 max 15 meters)

```
[i] Walking closer to [x,y] - distance is: 4788807 meters
[i] Walking closer to [x,y] - distance is: 4788801 meters
[i] Walking closer to [x,y] - distance is: 4788790 meters
[i] Walking closer to [x,y] - distance is: 4788786 meters
[i] Walking closer to [x,y] - distance is: 4788781 meters
[i] Walking closer to [x,y] - distance is: 4788767 meters
[i] Walking closer to [x,y] - distance is: 4788762 meters
```

**Pull request closed without merging**